features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,26 @@
+  When ("I visit the site") do
+    visit root_path
+  end
+  
+  Then ("I should see {string}") do |content|
+    expect(page).to have_content content
+  end
+  
+  When ("I click on {string} link") do |string|
+    click_on string
+  end
+  
+  When("I fill in {string} with {string}") do |field, input|
+   fill_in field, with: input
+  end
+  
+  When("I click on {string}") do |string|
+    click_on string
+  end
+  
+  Given("following users exist") do |table|
+    table.hashes.each do |user|
+        User.create!(user)
```

Comment:
I would like you to look into using factories rather than creating "real" instances

Reply:
We will work on this tomorrow morning asap, thanks

Reply:
Implemented factory "Test" for user

Reply:
You need to implement the Factory here in the step as well

Reply:
Ok done